packages/ember-htmlbars/tests/helpers/input_test.js

```diff
@@ -1,492 +0,0 @@
-    input.selectionEnd = 3;
-  });
-  run(null, set, controller, 'val', 'derp');
-
-  equal(view.$('input').val(), 'derp', 'updates text field after value changes');
-
-  equal(input.selectionStart, 3, 'cursor position was not lost');
-  equal(input.selectionEnd, 3, 'cursor position was not lost');
-});
-
-QUnit.test('input can be updated multiple times', function() {
-  equal(view.$('input').val(), 'hello', 'precondition - renders text field with value');
-
-  var $input = view.$('input');
-  var input = $input[0];
-
-  run(null, set, controller, 'val', '');
-  equal(view.$('input').val(), '', 'updates first time');
-
-  // Simulates setting the input to the same value as it already is which won't cause a rerender
-  run(function() {
-    input.value = 'derp';
-  });
-  run(null, set, controller, 'val', 'derp');
-  equal(view.$('input').val(), 'derp', 'updates second time');
-
-  run(null, set, controller, 'val', '');
-  equal(view.$('input').val(), '', 'updates third time');
-});
-
-
-QUnit.module('{{input type=\'text\'}} - static values', {
-  setup() {
-    commonSetup();
-
-    controller = {};
-
-    view = View.extend({
-      [OWNER]: owner,
-      controller: controller,
-      template: compile('{{input type="text" disabled=true value="hello" placeholder="Enter some text" name="some-name" maxlength=30 size=30 tabindex=5}}')
-    }).create();
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should insert a text field into DOM', function() {
-  equal(view.$('input').length, 1, 'A single text field was inserted');
-});
-
-QUnit.test('should become disabled if the disabled attribute is true', function() {
-  ok(view.$('input').is(':disabled'), 'The text field is disabled');
-});
-
-QUnit.test('input value is updated when setting value property of view', function() {
-  equal(view.$('input').val(), 'hello', 'renders text field with value');
-});
-
-QUnit.test('input placeholder is updated when setting placeholder property of view', function() {
-  equal(view.$('input').attr('placeholder'), 'Enter some text', 'renders text field with placeholder');
-});
-
-QUnit.test('input name is updated when setting name property of view', function() {
-  equal(view.$('input').attr('name'), 'some-name', 'renders text field with name');
-});
-
-QUnit.test('input maxlength is updated when setting maxlength property of view', function() {
-  equal(view.$('input').attr('maxlength'), '30', 'renders text field with maxlength');
-});
-
-QUnit.test('input size is updated when setting size property of view', function() {
-  equal(view.$('input').attr('size'), '30', 'renders text field with size');
-});
-
-QUnit.test('input tabindex is updated when setting tabindex property of view', function() {
-  equal(view.$('input').attr('tabindex'), '5', 'renders text field with the tabindex');
-});
-
-QUnit.test('specifying `on="someevent" action="foo"` triggers the action', function() {
```

Comment:
moved the deprecation to `@htmlbars Helpers test: {{input}}` : [``](https://github.com/emberjs/ember.js/pull/13194/files#diff-0caed08a992497ff104284ec6aea6d96R247)

Do we expect the `textField.trigger('focusin')` to trigger an action is our current test setup? It seems that `document.body` has no event handlers.